guides/source/generators.md

```diff
@@ -214,6 +214,7 @@ The next customization on the workflow will be to stop generating stylesheet and
 config.generators do |g|
   g.orm             :active_record
   g.template_engine :erb
+  g.stylesheets     false
   g.test_framework  :test_unit, fixture: false
```

Comment:
A later example adds the stylesheets after the test_framework option.
So I think it makes sense do do the same here.
It also makes it easier to see the changes if we append options to the end.
```suggestion
  g.test_framework  :test_unit, fixture: false
  g.stylesheets     false
```